integration-tests/integration_tests/integration_test_env.yaml

```diff
@@ -1,6 +1,6 @@
 #common
 MHS_LOG_FORMAT: "%(asctime)sZ | %(levelname)s | %(message)s | %(name)s | %(interaction_id)s | %(message_id)s | %(correlation_id)s | %(inbound_message_id)s"
-INTEGRATION_TEST_ASID: '918999198979'
+INTEGRATION_TEST_ASID: '' #put your ASID from OpenTest credentials
```

Comment:
It would be better to create an example yaml as for the other modules and .gitignore this one

Reply:
done